sympy/solvers/tests/test_solveset.py

```diff
@@ -1058,6 +1058,8 @@ def test_issue_9953():
     assert linsolve([ ], x) == S.EmptySet
 
 
+def test_issue_10534():
+    assert solveset_real(Piecewise((x, y<0), (x + 1, True)), x) == FiniteSet(-1,0)
```

Comment:
Now I don't think it should be in `piecewise ` , because check [this exaple](https://github.com/sympy/sympy/blob/master/sympy/solvers/tests/test_solveset.py#L630)
when we get solution for a `piecewise` equation then we get mostly interval and out is union of those 2,3 condition it don't return solution separately for each condition , right ?